MekHQ/src/mekhq/campaign/mission/AtBDynamicScenarioFactory.java

```diff
@@ -452,15 +452,10 @@ public static int generateForce(AtBDynamicScenario scenario, AtBContract contrac
     public static List<Entity> generateCivilianUnits(int num, Campaign campaign) {
         RandomUnitGenerator.getInstance().setChosenRAT("CivilianUnits");
         ArrayList<MechSummary> msl = RandomUnitGenerator.getInstance().generate(num);
-        List<Entity> retval = new ArrayList<>();
-        
-        List<Entity> entities = msl.stream().map(ms -> createEntityWithCrew("IND",
-                RandomSkillsGenerator.L_GREEN, campaign, ms))
-                .collect(Collectors.<Entity>toList());
-        retval.addAll(entities);
-        return retval;
+        List<Entity> entities = msl.stream().map(ms -> createEntityWithCrew("IND", RandomSkillsGenerator.L_GREEN, campaign, ms)).collect(Collectors.<Entity>toList());
+        return new ArrayList<>(entities);
```

Comment:
Why wrap this list in a new list?

Reply:
Was trying to keep same return type, however I've refactored it more so that it only needs to create the one ArrayList.